src/service/post.js

```diff
@@ -132,7 +133,7 @@ export const deletePost = async (postId) => {
 
 export const deleteAnonymousPost = async (postId) => {
   try {
-    const resApi = await api.delete(`/activity/anonymous/${postId}`);
+    const resApi = await anonymousApi.delete(`/activity/anonymous-v2/${postId}`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The endpoint for deleting anonymous posts has been changed. Make sure that the new endpoint `/activity/anonymous-v2/${postId}` is correct and the server-side implementation is ready to handle requests at this endpoint. 

```diff
- const resApi = await api.delete(`/activity/anonymous/${postId}`);
+ const resApi = await anonymousApi.delete(`/activity/anonymous-v2/${postId}`);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->